### PR TITLE
Feature/get all drivers returns List of drivers 

### PR DESCRIPTION
**Pull request by j-tanz**

### CODE REVIEW

**Comment by zxander-D** [comment]
needs work failing build

**Review by zxander-D — APPROVED** [review]
LGTM

File: src/main/java/com/galvanize/alpha/speedwaytrials/drivers/controller/DriversController.java [hunk 1]
```diff
@@ -13,17 +15,21 @@
 
 @RestController
 @RequestMapping("drivers")
+@AllArgsConstructor
```
Comment (davidgrieser) [inline src/main/java/com/galvanize/alpha/speedwaytrials/drivers/controller/DriversController.java:18]: This is odd in a controller. Did you mean to create a default constructor with the `DriverService` being passed in with **Depenedency Injection**?

**Review by davidgrieser — APPROVED** [review]
Seems functional, but the commented annotation is odd.

File: src/main/java/com/galvanize/alpha/speedwaytrials/drivers/controller/DriversController.java (hunk 1 above)
Comment (j-tanz) [reply src/main/java/com/galvanize/alpha/speedwaytrials/drivers/controller/DriversController.java:18]: @davidgrieser  Would you suggest we refactor to the boilerplate constructor syntax?  We were thinking that the AllArgsConstructor would force the DriverService dependency to be injected into the Controller, and that a future refactor was always possible if we needed more fine-grained handling of the controller instantiation.

**Pull request merged**